inc/spbc-settings.php

```diff
@@ -2373,7 +2373,7 @@ function spbc_field_scanner__show_accordion( $direct_call = false ) {
 	
 	$tables_files = array(
 		'critical'    => __('These files may not contain malicious code but they use very dangerous PHP functions and constructions! PHP developers don\'t recommend to use it and it looks very suspicious.', 'security-malware-firewall'),
-		'suspicious'  => __('Executable files which has been modified.', 'security-malware-firewall'),
+		'suspicious'  => __('Modified executable files found', 'security-malware-firewall'),
```

Comment:
Shouldn't the "found" word lead the sentence?

Reply:
Fixed.